### PR TITLE
RnD now starts at level 3 with most station research.

### PR DESCRIPTION
**Pull request by Geevies**
No need to sync it either, as it does so automatically after spawning.

image

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/modules/research/rdconsole.dm [hunk 1]
```diff
@@ -104,6 +127,8 @@ won't update every console in existence) but it's more of a hassle to do. Also,
 			S.setup()
 			break
 	SyncRDevices()
+	spawn(30)
```
Comment (JohnWildkins) [inline code/modules/research/rdconsole.dm]: could this be a timer instead?

File: code/modules/research/rdconsole.dm [hunk 2]
```diff
@@ -276,27 +301,7 @@ won't update every console in existence) but it's more of a hassle to do. Also,
 		else
 			griefProtection() //Putting this here because I dont trust the sync process
 			spawn(30)
```
Comment (JohnWildkins) [inline code/modules/research/rdconsole.dm]: same thing here

**Comment by Kaedwuff** [comment]
In my opinion, we should start with bluespace at 1. That's definitely sort of supposed to be the focus of research here, and we should encourage people to poke around at it more than the rest.

File: code/modules/research/rdconsole.dm (hunk 1 above)
Comment (Geevies) [inline code/modules/research/rdconsole.dm]:
```suggestion
	addtimer(CALLBACK(src, .proc/SyncTechs), 30)
```

File: code/modules/research/rdconsole.dm [hunk 3]
```diff
@@ -104,6 +127,8 @@ won't update every console in existence) but it's more of a hassle to do. Also,
 			S.setup()
 			break
 	SyncRDevices()
+	spawn(30)
+		SyncTechs()
```
Comment (Geevies) [inline code/modules/research/rdconsole.dm]:
```suggestion
```

File: code/modules/research/rdconsole.dm (hunk 2 above)
Comment (Geevies) [inline code/modules/research/rdconsole.dm]:
```suggestion
			addtimer(CALLBACK(src, .proc/SyncTechs), 30)
```

File: code/modules/research/rdconsole.dm [hunk 4]
```diff
@@ -276,27 +301,7 @@ won't update every console in existence) but it's more of a hassle to do. Also,
...
-								files.AddTech2Known(T)
-							for(var/datum/design/D in S.files.known_designs)
-								files.AddDesign2Known(D)
-							files.RefreshResearch()
-							server_processed = 1
-						if(!istype(S, /obj/machinery/r_n_d/server/centcom) && server_processed)
-							S.produce_heat()
-					screen = 1.6
-					updateUsrDialog()
+				SyncTechs()
```
Comment (Geevies) [inline code/modules/research/rdconsole.dm]:
```suggestion
```

**Comment by MarinaGryphon** [comment]
> In my opinion, we should start with bluespace at 1. That's definitely sort of supposed to be the focus of research here, and we should encourage people to poke around at it more than the rest.

This is silly. You get it to 3 in almost an instant; you print a beacon from the protolathe and the telescience console board from the circuit imprinter. It's trivial and the 2 levels makes no difference.

**Comment by Kaedwuff** [comment]
> > In my opinion, we should start with bluespace at 1. That's definitely sort of supposed to be the focus of research here, and we should encourage people to poke around at it more than the rest.
> 
> This is silly. You get it to 3 in almost an instant; you print a beacon from the protolathe and the telescience console board from the circuit imprinter. It's trivial and the 2 levels makes no difference.

Aight then.

**Pull request merged**